exec/internal/compile/compile.go

```diff
@@ -259,6 +294,7 @@ func Compile(disassembly []disasm.Instr) ([]byte, []*BranchTable) {
 			binary.Write(buffer, binary.LittleEndian, int64(0))
 			continue
 		case ops.BrIf:
+			emitMetadata(OpJmpNz, buffer.Len(), 18)
```

Comment:
please document what the special value `18` represents and/or how it was obtained.
perhaps consider making it a `const` like `instAndInt64Len` ?